nix/daedalus-bridge.nix

```diff
@@ -0,0 +1,33 @@
+{ target, pkgs, haskellPackages, system, crossSystem, jormungandr, jormungandr-cli }:
+
+let
+  commonLib = import ../lib.nix {};
+  pkgsCross = commonLib.getPkgs { crossSystem = pkgs.lib.systems.examples.mingwW64; };
+  cardano-shell = import commonLib.sources.cardano-shell { inherit system crossSystem; }; # todo, shell should accept target
+  jormungandrConfig = builtins.toFile "config.yaml" (builtins.toJSON commonLib.jormungandrLib.defaultJormungandrConfig);
+in pkgs.runCommandCC "daedalus-bridge" {
+  passthru = {
+    node-version = jormungandr.version;
+    wallet-version = haskellPackages.cardano-wallet-jormungandr.identifier.version;
+  };
+} ''
+  mkdir -pv $out/bin $out/config
+  cp ${jormungandrConfig} $out/config/jormungandr-config.yaml
+  cd $out/bin
+  cp ${haskellPackages.cardano-wallet-jormungandr.components.exes.cardano-wallet-jormungandr}/bin/cardano-wallet-jormungandr* .
+  cp ${cardano-shell.nix-tools.cexes.cardano-launcher.cardano-launcher}/bin/cardano-launcher* .
+  cp ${jormungandr-cli}/bin/jcli* .
+  #cp {nix-tools.cexes.cardano-node.cardano-node}/bin/cardano-node* .
```

Comment:
todo, accept the backend param, and select the right node and wallet binaries

Reply:
what is `cardano-node` here?

Reply:
new haskell node. Out of scope for this release.